### DISCUSSION TITLE
Documentation Error - NumberEntry element, wrong parameter name

### DISCUSSION THREAD

**Discussion by Adsecularis** [ctreffe/alfred#158] [General]
I hope the title makes sense. What I want to say is that I noticed an error in the [documentation for the NumberEntry element](https://alfredo3.psych.bio.uni-goettingen.de/docs/generated/alfred3.element.input.NumberEntry.html).
The first parameter listed is: "**decimals** – Accepted number of decimals (0 as default)." but that gave me an error:
TypeError: __init__() got an unexpected keyword argument 'decimals'

The solution was given to me by the autocomplete of PyCharm, namely that the parameter is now called: **ndecimals**

Also I am using Alfred 2.3.1 so I think it is reasonable to assume that the documentation is outdated and not my version of Alfred.

**Comment by Adsecularis** [comment]
Oh well, I just figured maybe I should have posted this as an issue, not in the discussions. So I don't mind if this discussions gets deleted or relocated. It seems I can't delete it myself.

**Comment by jobrachem** [comment]
You are right, there is a typo in the documentation. Thanks for pointing it out! 😊

**Reply by jobrachem** [reply]
The issue will be resolved in the next update. See #159. I'll close this discussion.
